packages/botonic-plugin-google-analytics-v4/src/types.ts

```diff
@@ -0,0 +1,25 @@
+import { Session, SessionUser } from '@botonic/core'
+
+export interface BotSession extends Session {
+  _botonic_action: string
+  _access_token: string
+  _hubtype_case_id?: string
+  user: SessionUser
+  lastMessageSentDate: string
+}
+
+export type GA4Options = {
+  getClientId?: () => string
+}
```

Comment:
The attributes of `apiSecret`, `measurementId` and `getUserId` should be added here.